agent/agent-plugins/redis-jedis-2.x/src/main/java/org/bithon/agent/plugin/redis/jedis2/interceptor/OnCommand.java

```diff
@@ -59,17 +58,15 @@ public InterceptionDecision before(AopContext aopContext) throws Exception {
 
         if (!ignoreCommands.contains(command)) {
             ITraceSpan span = TraceContextFactory.newSpan("jedis");
-            if (span == null) {
-                return InterceptionDecision.SKIP_LEAVE;
+            if (span != null) {
```

Comment:
this change is a good catch. Returning SKIP_LEAVE in previous release indeed cause inacurate metrics 👍